av1-rtp-spec.md

```diff
@@ -740,9 +740,14 @@ The Frame dependency structure includes a mapping between Decode targets and Cha
 
 #### A.6 Instantaneous Decidability of Decodability (IDD)
 
-Chains provide the IDD property. That is, the ability to decide immediately upon receiving the very first packet after packet loss, if any of the lost packets are required for the Decode target to remain decodable.
+The Instantaneous Decidability or Decodability (IDD) property is the ability to decide immediately whether a received or missed packet is needed to decode a received bitstream. This property allows an SFM to determine whether to forward a packet to one or more downstream endpoints or attempt to recover essential missed packets.

```

Comment:
This property allows an SFM to determine whether _it has_ to forward a packet to one or more downstream endpoints or attempt to recover essential missed packets.

or

This property allows an SFM to determine whether a received or missed packet must be forwarded to one or more downstream endpoints.